doc/symfony-cache-configuration.rst

```diff
@@ -143,6 +146,79 @@ one of ``client_ips`` or ``client_matcher``*.
 
   **default**: ``PURGE``
 
+
+Purge tags (cache invalidation using tags)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
```

Comment:
please add a `.. versionadded:: 2.1.0` that says that cache tagging for AppCache has been added in FOSHttpCache 2.1.0